recipes/recipe/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "hande" %}
+{% set version = "1.5" %}
+
+# Define build matrix for MPI vs. non-mpi
+# ensure mpi is defined (needed for conda-smithy recipe-lint)
+{% set mpi = mpi or 'nompi' %}
+{% set build = 1 %}
+{% if mpi == 'nompi' %}
+# prioritize 'nompi' variant via build number
```

Comment:
For priotization rather use track_festures, i.e. add a feature to the lower priority one.